### PR TITLE
Copy missing information from man pihole-FTL to pihole-FTL -h

### PR DESCRIPTION
**Pull request by yubiuser**
Copies a few missing information from `man pihole-FTL` into `pihole-FTL -h`.

https://github.com/pi-hole/pi-hole/blob/master/manpages/pihole-FTL.8

Man page for FTL is going to get removed for v6:  https://github.com/pi-hole/pi-hole/commit/f7b9d70054758137d60318e6df32ebb0d92987fe

### CODE REVIEW

**Comment by yubiuser** [comment]
Needs https://github.com/pi-hole/FTL/pull/1571 to be merged first

**Comment by yubiuser** [comment]
Rebased on latest `new/http`

File: src/args.c [hunk 1]
```diff
@@ -759,9 +759,10 @@ void parse_args(int argc, char* argv[])
 			printf("\t%s--list-dhcp6%s        List known DHCPv6 config options\n\n", green, normal);
 
 			printf("%sDebugging and special use:%s\n", yellow, normal);
-			printf("\t%sd%s, %sdebug%s            Enter debugging mode\n", green, normal, green, normal);
-			printf("\t%stest%s                Don't start pihole-FTL but instead\n", green, normal);
-			printf("\t                    quit immediately\n");
+			printf("\t%sd%s, %sdebug%s            Enter debugging mode:\n", green, normal, green, normal);
+			printf("\t							  Don't go into daemon mode + verbose logging");
```
Comment (DL6ER) [inline src/args.c:763]: Screenshot from 2023-06-10 13-11-51

Comment (DL6ER) [reply src/args.c:763]: There is are several line breaks missing here and the second sentence is way too much indented.

File: src/args.c [hunk 2]
```diff
@@ -798,9 +799,10 @@ void parse_args(int argc, char* argv[])
 			printf("\t                    interfaces\n");
 			printf("\t                    Append %s-x%s to force scan on all\n", cyan, normal);
 			printf("\t                    interfaces and scan 10x more often\n");
-			printf("\t%s--totp%s              Generate valid TOTP token for 2FA\n", green, normal);
+			printf("\t%s--totp%s          Generate valid TOTP token for 2FA\n", green, normal);
 			printf("\t                    authentication (if enabled)\n");
-			printf("\t%s-h%s, %shelp%s            Display this help and exit\n\n", green, normal, green, normal);
+			printf("\t%s--%s (options)	  Pass options to internal dnsmasq resolver\n", green, normal);
```
Comment (DL6ER) [inline src/args.c:804]: Screenshot from 2023-06-10 13-11-08

This kind of sticks out in a not so nice way

**Review by DL6ER — CHANGES_REQUESTED** [review]

File: src/args.c (hunk 2 above)
Comment (yubiuser) [reply src/args.c:818]: Screenshot at 2023-08-01 22-17-44

File: src/args.c (hunk 1 above)
Comment (yubiuser) [reply src/args.c:770]: Screenshot at 2023-08-01 22-18-00

**Review by DL6ER — APPROVED** [review]

**Pull request merged**